app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<section>
+  <%= form_for @task do |f| %>
+
+    <section>
+      <%= f.label :name %>
+      <%= f.text_field :name %>
+    </section>
+
+    <section>
+      <%= f.label :description %>
+      <%= f.text_field :description %>
+    </section>
+
+    <% if action_name == "Edit" %>
+      <section class = "input">
+        <%= f.label "Completed?" %>:
+        <%= f.select(:status, [true, false]) %>
```

Comment:
This doesn't seem to default to the current value of `status`.  

You could instead do:         
`         <%= f.select(:status, [true, false], {selected: @task.status}) %>`